vividus-util/build.gradle

```diff
@@ -1,6 +1,9 @@
 project.description = 'Vividus utilities'
 
-ext.jsonPathVersion = '2.6.0'
+ext {
+    jsonPathVersion = '2.6.0'
+    zxingVersion = '3.3.0'
```

Comment:
also BOM should be used when we import several dependencies and BOM is available

Reply:
I've moved dependencies, but I have not found a BOM for the ZXing platform.